### DISCUSSION TITLE
flex-search in maltilingual site

### DISCUSSION THREAD

**Discussion by roheclassic** [thuliteio/doks#783] [Q&A] [answered]
flex-search appears to search only for content in the primary language (only one language) in multilingual site.
And the primary language  seems to be determined by ```weight``` in language.toml.

Can I have flex-search match the display language?

**Comment by h-enk** [comment, marked as answer]
Would be part of multilingual improvements; I've create issue https://github.com/h-enk/doks/issues/790

**Reply by roheclassic** [reply]
Thanks!
